Lib/test/test_logging.py

```diff
@@ -1954,9 +1954,7 @@ def handle_request(self, request):
 
     def test_output(self):
         # The log message sent to the HTTPHandler is properly received.
-        logger = logging.getLogger("http")
-        root_logger = self.root_logger
-        root_logger.removeHandler(self.root_logger.handlers[0])
+        logger = logging.getLogger("httplogger")
```

Comment:
Why this name change?

Reply:
There is no specific reason for it. I'll just revert back.